### PR TITLE
UCS/CONFIG: Fix the name of alias

### PR DESCRIPTION
**Pull request by dmitrygladkov**
## What

Fix the name of an alias.

## Why ?

This prints wrong the name for aliases that has not only `UCX_` prefix.
Currently, it prints only `UCX_` prefix for all aliases, for example:
- good (real example for UCP context config):
`UCX_MAX_EAGER_LANES` alias for `UCX_MAX_EAGER_RAILS` real name
- bad (fictional example, but it is reproduced for PR that is under development):
`UCX_MAX_BCOPY` alias for `UCX_MM_SEG_SIZE` real name, but it has to be `UCX_MM_MAX_BCOPY`

## How ?

Pass the list of prefixes to the alias search routine and fill it. When a prefix is found, build the alias prefix string.

### CODE REVIEW

**Comment by dmitrygladkov** [comment]
@hoopoepg can take a look pls?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10135/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10138/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
@yosefe could you review pls?

**Comment by dmitrygladkov** [comment]
> what does it fix? can you please add/change a unit test to expose the problem?

@yosefe added the unit test (`test_config.test_config_print_opts` second part where `"TEST_"` is added) that exposes the problem that this PR solves

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10209/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10211/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10214/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10220/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10224/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
@yosefe hope the latest changes address your comments. could you review again pls?

File: test/gtest/ucs/test_config.cc [hunk 1]
```diff
@@ -221,25 +248,88 @@ class test_config : public ucs::test {
                                                               0);
             ASSERT_UCS_OK(status);
             return tmp;
-        }
+        }
```
[the remark this answers is not in the document]
Comment (dmitrygladkov) [reply test/gtest/ucs/test_config.cc]: removed

File: test/gtest/ucs/test_config.cc [hunk 2]
```diff
@@ -328,34 +427,32 @@ UCS_TEST_F(test_config, performance) {
     }
 
     /* Now test the time */
-    UCS_TEST_TIME_LIMIT(0.05) {
+    UCS_TEST_TIME_LIMIT(0.05){
```
[the remark this answers is not in the document]
Comment (dmitrygladkov) [reply test/gtest/ucs/test_config.cc]: fixed

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10258/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10265/ for details (Mellanox internal link).

**Comment by dmitrygladkov** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10287/ for details (Mellanox internal link).

**Pull request merged**